unstructured/documents/elements.py

```diff
@@ -651,19 +684,35 @@ def to_dict(cls):
 
 
 class Element(abc.ABC):
-    """An element is a section of a page in the document."""
+    """An element is a section of a page in the document.
```

Comment:
I would change this to: "An element is a portion of a document, generally a paragraph."

The term "section" has a specific meaning in partitioning, a "document-section" which is a "section-heading" and whatever text follows that heading until the next section heading.

Also, in DOCX it has a completely different meaning, basically a collection of pages with the same size and margins.

So better if we avoid that term when we're not talking about one of those two things.

Reply:
![image](https://github.com/Unstructured-IO/unstructured/assets/64484917/ff4e2313-6d3b-42e7-97a4-b9c3935e4d50)

🤔 yeah, "section" may sound misleading, it's fixed now